scripts/qa_crowbarsetup.sh

```diff
@@ -2908,6 +2908,12 @@ function onneutron_wait_for_neutron()
     fi
 }
 
+function onnovacontroller_wait_for_nova()
+{
+    get_novacontroller
+    wait_for 300 3 "ssh $novacontroller 'service openstack-nova-api status'" "nova-api service running state"
```

Comment:
@dirkmueller Done in https://github.com/SUSE-Cloud/automation/commit/7d72aafd9086910cc6951bdf57e573f7a69534fd